packages/notifier/test/test-publish-kit.js

```diff
@@ -277,22 +284,75 @@ test('durable publish kit upgrade trauma (full-vat integration)', async t => {
   const sub1 = await run('messageVat', [
     { name: 'pubsub', methodName: 'getSubscriber' },
   ]);
+  const spyName = 'receivePublicationRecord';
+  const directSubscriber = await run('makeRemotable', [
+    'directSubscriber',
+    { [spyName]: undefined },
+  ]);
+  const pub2Options = {
+    // onUpdate: makeMethodCallback(directSubscriber, spyName),
+    onUpdate: { target: directSubscriber, methodName: spyName, bound: [] },
```

Comment:
What's the reason to not use the maker?

Reply:
It doesn't work at the level outside the vats because `directSubscriber` isn't an object there.